hooks/pre-commit.d/cargo-audit

```diff
@@ -0,0 +1 @@
+cargo audit
```

Comment:
1. Detect if we are running on Travis.
2. If on Travis, return success immediately unless the `--force` option is used.